_layouts/home.html

```diff
@@ -2,6 +2,23 @@
 layout: archive
 ---
 
+{% if page.url == "/" %}
```

Comment:
This doesn't look good to me. Perhaps it should be a page-level switch instead of hard-coded "root path"